src/components/Card.tsx

```diff
@@ -5,17 +5,23 @@ import Likes from "./Likes";
 import Comments from "./Comments";
 import Views from "./Views";
 
-const Card = () => {
-	const name = "Muhammad Saleh";
-	const date = "December 06, 2024";
-	const views = 123;
-	const comments = 456;
-	const likes = 789;
-
-	const handleExplainMore = () => {
-		// Code to display more project information
-	};
+interface CardProps {
+	name: string;
+	date: string;
+	views: number;
+	comments: number;
+	likes: number;
+	onExploreMore: () => void;
+}
```

Comment:
Replacing handleExplainMore with onExploreMore was a good idea, but make sure onExploreMore is fully defined in the main logic, as it looks like the handler isn’t here in the component.

Reply:
Yes, it's defined in the parent component